components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -18,10 +18,13 @@ package controllers
 import (
 	"context"
 	"io/ioutil"
-	netv1 "k8s.io/api/networking/v1"
 	"strings"
 	"time"
 
+	"github.com/onsi/gomega/format"
+	netv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
there has been a mistake cherry-picking this; the original commit also deleted the duplicate of this line at line 32 in this file, but this commit left it there